spec/views/statuses/show.html.haml_spec.rb

```diff
@@ -8,17 +8,18 @@
     allow(view).to receive(:local_time)
     allow(view).to receive(:local_time_ago)
     assign(:instance_presenter, InstancePresenter.new)
-  end
 
-  it 'has valid opengraph tags' do
-    alice  = Fabricate(:account, username: 'alice', display_name: 'Alice')
-    status = Fabricate(:status, account: alice, text: 'Hello World')
     Fabricate(:media_attachment, account: alice, status: status, type: :video)
 
     assign(:status, status)
     assign(:account, alice)
     assign(:descendant_threads, [])
+  end
 
+  let(:alice) { Fabricate(:account, username: 'alice', display_name: 'Alice') }
+  let(:status) { Fabricate(:status, account: alice, text: 'Hello World') }
```

Comment:
Wouldn't this be more readable if this were placed before the `before` block?

Reply:
Pushed commit to move before before.